brewchat/src/main/java/com/example/brewchat/activities/LoginActivity.java

```diff
@@ -68,8 +79,14 @@ public void onEvent(UserLoggedEvent event) {
         startActivity(intent);
     }
 
+    @SuppressWarnings("unused")
+    public void onEvent(UserSignedUpEvent event) {
+        Toast.makeText(this,"Account Created!",Toast.LENGTH_LONG).show();
```

Comment:
All captions should be extracted to ´Strings´ file during a feature development. Latter refactors may never come.